src/helm/benchmark/run_expander.py

```diff
@@ -326,20 +320,23 @@ class NumOutputsRunExpander(ReplaceValueRunExpander):
     }
 
 
-class ModelRunExpander(ReplaceValueRunExpander):
+class ModelDeploymentRunExpander(ReplaceValueRunExpander):
```

Comment:
I think `ModelRunExpander` should stay largely the same. `ModelDeploymentRunExpander` should inherit directly from `ReplaceValueRunExpander` and it shouldn't do any kind of value group expansion.

The reason this does not work is because (1) if you set `model_deployment=text` and (2) `meta/llama-2-7b` has two deployments, `huggingface/llama-2-7b` and `together/llama-2-7b`, then _both_ model deployments get run, which is not really what is intended.

I also think we should get rid of the tokenizer-based expansions (don't think it is still being used).

Reply:
See the other comments. There is only a `ModelDeploymentRunExpander` and it handles all cases.

Reply:
If you do `boolq:model=meta/llama-7b`, after the run expanders , `adapter.model` is `"huggingface/llama-7b"`, and `adapter.model_deployment` is `None`. In `alter_run_specs()`, `get_model_deployment(run_spec.adapter_spec.model_deployment)` will fail because that `adapter_spec.model_deployment` is `None`. Or am I missing something?

Additionally, I believe the above example will also have the the updated run name of `boolq:model=huggingface/llama-7b`, which is odd.